Doc/source/feaLib/index.rst

```diff
@@ -1,17 +1,40 @@
-######
-feaLib
-######
+#########################################
+feaLib: Read/write OpenType feature files
+#########################################
 
-.. toctree::
-   :maxdepth: 1
+fontTools' ``feaLib`` allows for the creation and parsing of Adobe
+Font Development Kit for OpenType feature (.fea) files. The syntax
```

Comment:
Should file name extensions be rendered as code? The documentation is inconsistent.

Reply:
I'll say yes, and make a note to fix up the inconsistencies.